cmake/dependencies/imath.cmake

```diff
@@ -29,6 +29,7 @@ SET(_install_dir
 SET(_include_dir
     ${_install_dir}/include/Imath
 )
+SET(RV_DEPS_IMATH_ROOT_DIR ${_install_dir})
```

Comment:
IMath can be made to use the new Macros (from Nicolas), less duplicated code and a nice Clean Target.